samples/cpp/benchmark_app/utils.cpp

```diff
@@ -110,6 +117,17 @@ std::vector<float> split_float(const std::string& s, char delim) {
     return result;
 }
 
+void check_system_configuration() {
```

Comment:
Can the same be done for Python?

Reply:
So far I've only found a way to execute a powershell command to do the same, but if needed I could've just created a binding I guess. But before that I wanted to get your input about this solution. The README is now updated with the warning message!